src/breadcrumb/breadcrumb-item.interface.ts

```diff
@@ -12,6 +12,16 @@ export interface BreadcrumbItem {
 	 * Href for the breadcrumb item.
 	 */
 	href: string;
```

Comment:
Optional suggest, since we have the `route` key now, `href` should be optional

```suggestion
	href?: string;
```

Reply:
Yea, I was thinking about the same thing but wasn't sure about it